README.adoc

```diff
@@ -0,0 +1,143 @@
+= Istio Circuit Breaker mission
+
+== Purpose
+Showcase Istio's Circuit Breaker via a (minimally) instrumented Spring Boot application
+
+== Prerequisites
+
+- Openshift 3.9 cluster
+- Istio 0.7.1 with authentication installed on the aforementioned cluster.
+To install Istio simply follow one of the following docs:
+    * https://istio.io/docs/setup/kubernetes/quick-start.html
+    * https://istio.io/docs/setup/kubernetes/ansible-install.html
+- Enable automatic sidecar injection for Istio
+  * See https://istio.io/docs/setup/kubernetes/sidecar-injection.html[this] for details
+- Login to the cluster with the admin user
+
+NOTE: The `istiooc cluster up --istio=true` command from https://github.com/openshift-istio/origin/releases/[this project] perfectly satisfy the above requirements.
+
+== Environment preparation
+
+```bash
+    oc new-project circuit-breaker-mission
+```
+*CAUTION*:
+In order for Istio automatic sidecar injection to work properly the following Istio configuration needs to be in place:
+
+. The `policy` field is set to `disabled` in the `istio-inject` configmap  of the `istio-system` namespace
+. The `istio-sidecar-injector` `MutatingWebhookConfiguration` should not limit the injection to properly labeled namespaces
+. Expose services and Istio ingress
```

Comment:
This is not needed. Why did you add this?

Reply:
Why is it incorrect? 
If I am reading it correctly, it is correct and it's purpose is to make the user aware of the Istio configuration that needs to be in place for this booster to work

Reply:
I'm talking about exposing the services and Istio ingress. There is no need to do so for this booster.

Reply:
Cool, thanks for the clarification!

Reply:
I added this because all of the boosters should have the same Istio configuration docs. Even if this step is not necessary for *this* booster, if they go to try another booster and think they've set up Istio correctly, it will fail. Granted, none of this really matters because users should be using Istiooc anyway.